src/dragonball/src/dbs_virtio_devices/src/balloon.rs

```diff
@@ -906,8 +910,10 @@ pub(crate) mod tests {
             let kvm = Kvm::new().unwrap();
             let vm_fd = Arc::new(kvm.create_vm().unwrap());
             let resources = DeviceResources::new();
+            let address_space = create_address_space();
             let config = VirtioDeviceConfig::<Arc<GuestMemoryMmap<()>>>::new(
                 Arc::new(mem),
+                address_space,
```

Comment:
nit: could you separate all addreess_space related change to a different commit?

Reply:
I've slipt that changes into a separate commit.